src/script/page/LeftSidebar/panels/Conversations/state.ts

```diff
@@ -68,22 +69,51 @@ export enum SidebarTabs {
   PREFERENCES,
 }
 
+export const SidebarOpenStatus = {
+  OPEN: 'OPEN',
+  CLOSED: 'CLOSED',
+  MANUAL_OPEN: 'MANUAL_OPEN',
+  MANUAL_CLOSED: 'MANUAL_CLOSED',
+} as const;
+
+export type SidebarOpenStatus = (typeof SidebarOpenStatus)[keyof typeof SidebarOpenStatus];
+
 export interface SidebarStore {
-  isOpen: boolean;
-  setIsOpen: (isOpen: boolean) => void;
-  toggleIsOpen: () => void;
+  openStatus: SidebarOpenStatus;
```

Comment:
`openStatus` is never set to `MANUAL_...` statuses, right? - then I'd set it to `boolean`